auditlog/tests/test_auditlog.py

```diff
@@ -368,3 +368,173 @@ def setUp(self):
     def tearDown(self):
         self.groups_rule.unlink()
         super(TestAuditlogFullCaptureRecord, self).tearDown()
+
+
+class AuditLogRuleTestCommon(TransactionCase):
+    def setUp(self):
+        super(AuditLogRuleTestCommon, self).setUp()
+        # get Contact model id
+        self.contact_model_id = (
+            self.env["ir.model"].search([("model", "=", "res.partner")]).id
+        )
+
+        # get phone field id
+        self.fields_to_exclude = (
+            self.env["ir.model.fields"]
+            .search([("model", "=", "res.partner"), ("name", "=", "phone")])
+            .id
+        )
+
+        # get user id
+        self.user = (
+            self.env["res.users"]
+            .with_context(no_reset_password=True)
+            .create(
+                {
+                    "name": "Test User",
+                    "login": "testuser",
+                }
+            )
+        )
+        self.user_2 = (
+            self.env["res.users"]
+            .with_context(no_reset_password=True)
+            .create(
+                {
+                    "name": "Test User2",
+                    "login": "testuser2",
+                }
+            )
+        )
+
+        self.users_to_exclude = self.user.id
+
+        # creating auditlog.rule
+        self.auditlog_rule = self.env["auditlog.rule"].create(
+            {
+                "name": "testrule 01",
+                "model_id": self.contact_model_id,
+                "log_read": True,
+                "log_create": True,
+                "log_write": True,
+                "log_unlink": True,
+                "log_type": "full",
+                "capture_record": True,
+            }
+        )
+
+    def test_01_AuditlogFull(self):
```

Comment:
would be nice to split this tests in smaller tests

Reply:
added smaller test splits